Makefile

```diff
@@ -58,8 +62,10 @@ run: | build run-kernel-gateway
 		-e HTTPS_PORT=$(HTTPS_PORT) \
 		-e HTTPS_KEY_FILE=$(HTTPS_KEY_FILE) \
 		-e HTTPS_CERT_FILE=$(HTTPS_CERT_FILE) \
-		-e KERNEL_GATEWAY_URL=http://$(KG_CONTAINER_NAME):8888 \
+		-e KERNEL_GATEWAY_URL=$(KERNEL_GATEWAY_URL) \
+		-e KERNEL_CLUSTER_URL=$(KERNEL_CLUSTER_URL) \
 		--link $(KG_CONTAINER_NAME):$(KG_CONTAINER_NAME) \
+		--link $(TMPNB_PROXY_CONTAINER_NAME):$(TMPNB_PROXY_CONTAINER_NAME) \
 		$(DASHBOARD_IMAGE_NAME) $(CMD)
 
 run-debug: CMD=start-debug
```

Comment:
I'll change the make targets to start the node app with **either** the kernel gateway or tmpnb.

Regardless of the make targets, the code has to check for `CLUSTER_URL`, then `GATEWAY_URL`, then #fail.